test/com/facebook/buck/android/testdata/android_instrumentation_test_integration_test/apps/myapp_test/BUCK

```diff
@@ -0,0 +1,14 @@
+android_instrumentation_apk(
+  name = 'app_test',
+  manifest = 'AndroidManifest.xml',
+  apk = '//apps/myapp:app-exo',
```

Comment:
suggestion for another version (we don't have to do this in this pull request):

we should be able to update the AndroidManifest to automatically generate the instrumentation/targetPackage field based on the properties here so we can remove that piece of redundancy. I think gradle does something like this, but let's leave this to another change.